cloudflare/resource_cloudflare_access_application_test.go

```diff
@@ -424,6 +424,33 @@ resource "cloudflare_access_application" "%[1]s" {
 `, rnd, zoneID, domain, accountID)
 }
 
+func testAccCloudflareAccessApplicationConfigWithGatewayRules(rnd, zoneID, domain string, accountID string) string {
```

Comment:
@AkemiDavisson looks like this is unused and we don't have any test coverage for these new fields. do you mind updating this?

Reply:
Updated! 👍